metadata.json

```diff
@@ -47,10 +47,6 @@
   ],
   "requirements": [
     {
-      "name": "pe",
-      "version_requirement": "3.x"
-    },
-    {
       "name": "puppet",
       "version_requirement": "3.x"
```

Comment:
It works with puppet 4 too?  Maybe replace with something like
`"version_requirement": ">= 3.4.0 < 5.0.0"`